rootfs/usr/bin/playtronos-systemreport

```diff
@@ -47,6 +49,23 @@ handle_options() {
       usage
       exit 0
       ;;
+    -o|--out)
+      OUTPUT_PATH="$2"
```

Comment:
I might be missing something, but why `$2`? I suspect that will not work in the general case when multiple options are specified.

We should probably use the same pattern as the `--report` option parsing is using?

i.e. using `has_argument` and `extract_argument` with `$@` ?